### PR TITLE
Load user's address from DB

### PR DESCRIPTION
**Pull request by uxengine**
These backend changes are needed for
https://github.com/orbiting/republik-frontend/pull/7
which allows editing a user's personal data and (some) profile data.

FYI, we expose isPrivate on the API because on the account page we want to be able to edit the publicUser data and the isPrivate flag at the same time.

### CODE REVIEW

File: graphql/resolvers/_RootQuerys/me.js [hunk 1]
```diff
@@ -30,7 +28,10 @@ module.exports = async (_, args, { user }) => {
   return {
     ...user,
     name,
+    address: await pgdb.public.addresses.findOne({ id: user.addressId }),
```
Comment (patte) [inline graphql/resolvers/_RootQuerys/me.js]: This calls for a separate User/-address resolver. The advantage is that this DB roundtrip can be saved if the field is not requested. Checkout graphql/resolvers/Comment.js

**Review by patte — APPROVED** [review]

File: graphql/resolvers/_RootQuerys/me.js (hunk 1 above)
Comment (uxengine) [reply graphql/resolvers/_RootQuerys/me.js]: Done. LGTY?

File: graphql/resolvers/User.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  name (user) {
+    return [user.firstName, user.lastName].join(' ')
+  },
```
Comment (patte) [inline graphql/resolvers/User.js]: user.name is required in many places, thats why we have lib/factories/createUser. I would use this here.

**Comment by uxengine** [comment]
Updated factory now as just discussed.

**Pull request merged**